code/modules/client/preferences_savefile.dm

```diff
@@ -56,6 +56,9 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 	if(current_version < 33)
 		toggles |= SOUND_ENDOFROUND
 
+	if(current_version < 36)
+		clientfps = "Smooth(40 FPS)"
```

Comment:
this can't be test merged if it's overwriting someone's fps, you have to either convert, or use a new value and deprecate the old one

Reply:
fixed